Dependencies/UrlLib/Source/Unix/UrlRequest.cpp

```diff
@@ -174,7 +174,58 @@ namespace UrlLib
             if (curl)
             {
                 data.clear();
-                curl_easy_setopt(curl, CURLOPT_URL, m_url.data());
+
+                CURLUcode rc;
+                CURLU *url = curl_url();
```

Comment:
nit in a few places:

```suggestion
                CURLU* url = curl_url();
```